libero-config/config--libero-default.js

```diff
@@ -50,7 +50,7 @@ config.data.color.attention = Color('#cf0c4e');
 config.data.color.warning = Color('#e65100');
 
 config.data.font = { letterspacing: {} };
-config.data.font.primary = '"Noto Serif", serif';
+config.data.font.primary = '"Noto Serif", "Noto Sans", serif';
```

Comment:
Not sure what the best course of action is here. Some scripts (eg Arabic) don't have serif forms. Maybe can alias `Noto Sans` as `Noto Serif` instead in those cases?